src/main/java/org/libtorrent4j/Vectors.java

```diff
@@ -137,4 +138,35 @@ public static byte_vector ascii2byte_vector(String s) {
 
         return arr;
     }
+
+    public static byte_array_32 bytes2byte_array_32(byte[] arr) {
```

Comment:
Codacy found an issue: [The static method name 'bytes2byte_array_32' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5867294)